content/blog/2018/06/2018-06-26-lessons-java10-hackathon.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "What I learnt at the Jenkins & Java 10+ Hackathon"
+tags:
+- events
+- community
+- developer
+- java10
+- java11
+author: tracymiranda
+---
+
+Last week I participated in the link:/blog/2018/06/08/jenkins-java10-hackathon/[Jenkins & Java 10 Online Hackathon].
+It was my first Jenkins hackathon and I roped in link:https://github.com/jonahgraham[Jonah Graham] to do some pair-programming.
+The hackathon, organized by link:https://github.com/oleg-nenashev[Oleg Nenasheva], featured JDK Project Jigsaw committers Mandy Chung and Paul Sandoz, as well as Jenkins creator Kohsuke Kawaguchi. 
```

Comment:
s/Oleg/Jenkins Events Team maybe

Reply:
Will remove that phrase and link Oleg at first use later on.